release_notes/addtl-release-notes.adoc

```diff
@@ -0,0 +1,99 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="addtl-release-notes"]
+= Additional release notes
+include::_attributes/common-attributes.adoc[]
+include::_attributes/attributes-microshift.adoc[]
+include::_attributes/servicebinding-document-attributes.adoc[]
+:context: addtl-release-notes
+
+toc::[]
+
+Release notes for additional related components and products not included in the core xref:../release_notes/ocp-release-notes.adoc#ocp-release-notes[{product-title} {product-version} release notes] are available in the following documentation.
+
+[IMPORTANT]
+====
+The following release notes are for downstream Red Hat products only; upstream or community release notes for related products are not included.
+====
+
+[cols="1a,1a,1a",frame=none,grid=none]
+|===
+|A::
+xref:../networking/aws_load_balancer_operator/aws-load-balancer-operator-release-notes.adoc#aws-load-balancer-operator-release-notes[AWS Load Balancer Operator]
+
+|B::
+link:https://docs.redhat.com/en/documentation/builds_for_red_hat_openshift/1.1/html/about_builds/ob-release-notes#ob-release-notes[{builds-v2title}]
```

Comment:
@adellape and @bergerhoffer 

- Should this not be under O, as it is part of OpenShift ?
- and the name should start with a small case and not capital B, it must be `builds for Red Hat OpenShift`, correct?

Reply:
https://github.com/openshift/openshift-docs/pull/84519#pullrequestreview-2418203616